processing/src/main/java/io/druid/segment/DoubleColumnSelector.java

```diff
@@ -26,5 +26,32 @@
 public interface DoubleColumnSelector extends ColumnValueSelector, HotLoopCallee
 {
   @CalledFromHotLoop
-  double get();
+  @Override
+  double getDouble();
```

Comment:
What do you think about renaming these methods as ```asDouble()```, ```asFloat()```, and ```asLong()```, and adding another method ```get()``` which calls ```getDouble()``` by default? It's more intuitive to me.
Same for other ColumnValueSelectors.